runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -1509,12 +1553,12 @@ private static String longBossName(String boss)
 			// The Gauntlet
 			case "gaunt":
 			case "gauntlet":
-				return "Gauntlet";
```

Comment:
youll need to add these as cases for each gauntlet because thats what the kc message has

Reply:
I'm not sure i follow what you are saying alex, I need the `The ` prefixed onto the long boss name result for the gauntlet values. This is because the HiscoreSkill has `THE_GAUNTLET` and `THE_CORRUPTED_GAUNTLET` and we use `HiscoreSkill.valueOf()` to grab the correct enum value.

I can remove `THE_` from that enum but it'll require changing multiple things, including renaming two images, so this seemed like the cleanest solution.

Reply:
the the will be fine if thats what it needs to return, you just also need to have a case for what they currently are because the kc message that the game gives you when you complete one doesnt have a `the` in it.